Description: Patch to modify Tools/LinuxScripts/rcon script for use in a Debian based distro.
===================================================================
--- ./Tools/LinuxScripts/rcon~	2007-05-14 23:37:17.000000000 -0400
+++ ./Tools/LinuxScripts/rcon	2007-05-14 23:37:17.000000000 -0400
@@ -22,6 +22,10 @@
     Tony Jackson can be contacted at tonyj@cooldark.com
 	
 =end
+=begin
+	This script has been patched from its original source for distribution
+	with Debian systems.
+=end
 
 require 'socket'
 
@@ -103,10 +107,10 @@
 	puts 'RCon utility for Alien Arena servers'
 	puts ''
 	puts '  Usage:'
-	puts '  rcon <server:port> password <commandstring>'
+	puts '  alien-arena-server -r <server:port> password <commandstring>'
 	puts '  Optional arguments:'
 	puts ''
-	puts '  --help  This message'
+	puts '  alien-arena-server -r --help  This message'
 	puts ''
 	exit
 end
